primer/index.bs

```diff
@@ -289,53 +293,84 @@ That URL might look a little complex, but it's essentially a request to
     - `openid` is a scope that is needed to verify Alice's identity.
     - `webid` is required by the Solid-OIDC specification to denote a WebID login.
     - `offline_access`: Is required to get a refresh token.
-- `client_id=https%3A%2F%2Fdecentphotos.example%2Fclient_id`: Usually the client id of a Solid
-    application is the app's URI (in our case `https://decentphotos.example/client_id`) as seen here.
+- `client_id=https%3A%2F%2Fdecentphotos.example`: Usually the client id of a Solid
+    application is the app's URI (in our case `https://decentphotos.example`) as seen here.
 - `code_challenge_method=S256`: Tells the OP that our code challenge was transformed using SHA-256.
 - `code_challenge=HSi9dwlvRpNHCDm-L8GOdM16qcb0tLHPZqQSvaWXTI0`: The [=code challenge=] we generated before.
 
 Note: If the app doesn't have a URI, you can either register an app using static registration
```

Comment:
> now I'm farily certain that the Federation spec only means to add that to the .well-known/openid-federation config

For an OP that is _not_ part of a federation, how would a client know that the OP supports `automatic` registration? Unless you are suggesting that all such OPs must also participate in a federation? I had not been making that assumption.

Reply:
Not sure how that is related to (not) mentioning static registration? Did you mean to respons to the other conversation on `client_registration_types_supported`, @acoburn?

Reply:
To answer to your question: I was assuming that a Solid-OIDC compliant OP would always adhere to the OpenID Federation spec, regardless of it being part of an actual federation or not; so every OP would _always_ have a `.well-known/openid-federation` with the `client_registration_types_supported` list. Since that list has no defined meaning in OpenID Connect, however, I don't think it has any role in `.well-known/oidc-configuration`. But that's just how I initially interpreted it; might be wrong.

Reply:
Aside: @woutermont this is in a different thread because the GH interface, for some inexplicable reason, will not allow me to comment on the relevant thread.

One perspective is that, if a Solid-OIDC OP is going to support the processing of RP Federation trust chains, it would make sense to also require the OP to participate in a Federation. After all, the OP is already implementing most of OIDC Federation, it could sure support the remaining APIs easily enough.

The other perspective is that the specific need on the part of Solid-OIDC is to be able to uniquely and securely identify RPs. Adding requirements to OPs above and beyond the needs of Solid only adds a burden to implementers.

I suspect that many Solid-OIDC OP implementations will also participate in a Federation, but I am wary of adding requirements that are not strictly necessary. It will be important to ensure that there is nothing in the text of the specification that _prevents_ an IdP implementer from adding full OP support for federation.

Reply:
I totally agree, @acoburn! We should not make it any more difficult for OPs than necessary.

I just don't see how my suggestion (not including `client_registration_types_supported` in `.well-known/oidc-configuration`) would do that. As I see it, the only thing we really require from OPs is that they support Automatic Registration, and declare that in their `.well-known/openid-federation` statement. There is no need to also declare it in  `.well-known/oidc-configuration`, because it has no meaning for clients reading that configuration. 

Unless I am mistaken, this entails no other requirements from the OP.